x/gamm/keeper/pool_service_test.go

```diff
@@ -483,43 +490,50 @@ func (suite *KeeperTestSuite) TestExitPool() {
 	}
 
 	for _, test := range tests {
-		suite.SetupTest()
+		suite.Run(test.name, func() {
```

Comment:
Note to reviewers: the diff is large because of indentation change due to `suite.Run`